File: JULIA/Cap03/c03ex03.jl
# c03ex03.jl

print("Entre quantidade de horas trabalhadas ..: ")
ht = parse(Float64, readline())

print("Entre valor do salario-hora ............: ")
vh = parse(Float64, readline())

print("Entre valor do percentual de desconto ..: ")
pd = parse(Float64, readline())

sb = ht * vh
td = (pd / 100.0) * sb

sl = sb - td

println("\nSalario Bruto ....: $(round(sb, digits=2))")
println("Salario Liquido ..: $(round(sl, digits=2))")

print("\nTecle <Enter> para encerrar... ")
readline()

File: SCALA/Cap03/c03ex03.scala
// c03ex03.scala

import scala.io.StdIn.{readDouble, readLine}

object Main {
  def main(args: Array[String]): Unit = {
    var HT, VH, PD, SB, TD, SL: Double = 0.0

    print("Entre quantidade de horas trabalhadas ..: ")
    HT = readDouble()

    print("Entre valor do salario-hora ............: ")
    VH = readDouble()

    print("Entre valor do percentual de desconto ..: ")
    PD = readDouble()

    SB = HT * VH
    TD = (PD / 100) * SB
    SL = SB - TD

    printf("\nSalario Bruto ....: %.2f\n", SB)
    printf("Salario Liquido ..: %.2f\n", SL)

    print("\nTecle <Enter> para encerrar... ")
    readLine()
  }
}
